collections/openapi.yaml

```diff
@@ -77,20 +69,6 @@ paths:
                     rel: data
                     type: application/json
                     title: Information about the feature collections
-  '/conformance':
```

Comment:
Removing due to inheritance from Core, I guess?

Reply:
Correct.